docs/en/week08/08-3.md

```diff
@@ -0,0 +1,198 @@
+- Next, we will sample $\boldsymbol{z}$ from the above distribution parametrized by the encoder; specifically, $\mathbb{U}(\boldsymbol{z})$ and $\mathbb{V}(\boldsymbol{z})$ are passed into a **sampler** to generate the latent variable $\boldsymbol{z}$.
+- Finally, $\boldsymbol{z}$ is passed into the decoder to generate $\tilde{\boldsymbol{y}}$.
+    - The decoder will be a function from $\mathcal{Z}$ to $\mathbb{R}^{n}$: $\boldsymbol{z} \mapsto \boldsymbol{\tilde{y}}$.
+
+In fact, for basic autoencoder, we can think of $\boldsymbol{h}$ as just the vector $\mathbb{U}(\boldsymbol{z})$ in the VAE formulation. In short, the main difference between VAEs and AEs is that VAEs have a good latent space that enables generative process.
+
+
+**The VAE objective (loss) function**
+
+<center>
+<img src="{{site.baseurl}}/images/week08/08-3/VAEloss.png" /><br>
+<b>Fig. 5</b>: Mapping from input space to latent space
+</center>
+
+First, we encode from input space (left) to latent space (right), through encoder and noise. Next, we decode from latent space (right) to output space (left). To go from the latent to input space (the generative process) we will need to either learn the distribution (of the latent code) or enforce some structure. In our case, VAE enforces some structure to the latent space.
+
+As usual, to train VAE, we minimize a loss function. The loss function is therefore composed of a reconstruction term as well as a regularization term.
+- The reconstruction term is on the final layer (left side of the figure). This corresponds to $C(\boldsymbol{y}, \tilde{\boldsymbol{y}})$ in the figure.
+- The regularization term is on the latent layer, to enforce some specific Gaussian structure on the latent space (right side of the figure). We do so by using a penalty term $D_{KL}(\mathcal{N}(\boldsymbol{\mu}, \boldsymbol{v}) || \mathcal{N}(\boldsymbol{0}, \boldsymbol{1}))$. Without this term, VAE will act like a basic autoencoder, which may lead to overfitting, and we won't have the generative properties that we desire.
+
+**Discussion on sampling $\boldsymbol{z}$ (reparameterisation trick)**
+
+How do we sample from the distribution returned by the encoder in VAE? From above we know that, we sample from the Gaussian distribution, in order to obtain $\boldsymbol{z}$. However, this is problematic, because when we do gradient descent to train the VAE model, we don't know how to do backpropagation through the sampling module.
+
+Instead, we use the *reparameterization trick* to "sample" $\boldsymbol{z}$. We use $\boldsymbol{z} = \mathbb{U}(\boldsymbol{z}) + \boldsymbol{\epsilon} \odot \sqrt{\mathbb{V}(\boldsymbol{z})}$ where $\epsilon\sim \mathcal{N}(\boldsymbol{0}, \boldsymbol{I}_d)$. In this case, backpropagation in training is possible. Specifically, the gradients will go through the (element-wise) multiplication and addition in the above equation.
```

Comment:
This is wrong. Please, use *this year* notation.
We are making *new* transcriptions because the content is *new*.

Reply:
fixed